mono/metadata/locales.c

```diff
@@ -294,20 +300,20 @@ construct_culture (MonoCultureInfo *this_obj, const CultureInfoEntry *ci)
 }
 
 static MonoBoolean
-construct_region (MonoRegionInfo *this_obj, const RegionInfoEntry *ri)
+construct_region (MonoRegionInfoHandle* this_obj, const RegionInfoEntry *ri)
```

Comment:
Hey guys,

 A handle is either a pointer or an index to the backing storage, they are not raw pointers to the objects.

This way, copying the handle value is perfectly fine and we ensure we won't have any ABI issues on where they live.